db/schema.rb

```diff
@@ -589,8 +589,6 @@
     t.string "direct_otp"
     t.datetime "direct_otp_sent_at"
     t.string "unique_session_id"
-    t.string "email_fingerprint", default: "", null: false
-    t.text "encrypted_email", default: "", null: false
     t.text "encrypted_phone"
```

Comment:
👀 oooo definitely not for this PR but I suspect this is another column we could drop because we moved to `phone_configurations`

Reply:
Definitely.  Likewise I think `totp_timestamp` could be dropped